alpha/books/RayTracingInOneWeekend.html

```diff
@@ -561,10 +564,11 @@
 
 (If you're wondering why we don't just use `aspect_ratio` when computing `viewport_width`, it's
 because the _actual_ aspect ratio is the integer-based ratio of image width over image height, which
-will yield zero for values less than one.)
+_may_ not be the same as `aspect_ratio` because `image_height` may have off round-off error and may
+have been set to 1.)
```

Comment:
This over focuses on the case where image_height is small; it's more general than that.

Consider the case where aspect_ratio is 4/3, and we're given an image_width of 25. In this case, the corresponding image_height would be 25/(4/3) = 18, yielding a true aspect ratio of 25/18 = 1.39.

Reply:
I mean, I tried to address that, that's what "may have off round-off error" us trying to get across. I'll try again